tests/NHSD.BuyingCatalogue.Ordering.Api.UnitTests/Validation/CreateOrderItemModelValidatorTests.cs

```diff
@@ -421,15 +421,54 @@ static CreateOrderItemModelValidatorTests()
         }
 
         [Test]
-        [CommonInlineAutoData(nameof(CatalogueItemType.AdditionalService))]
-        [CommonInlineAutoData(nameof(CatalogueItemType.AssociatedService))]
-        public static void Validate_DeliveryDateIsNull_IsNotSolution_DoesNotHaveError(
-            string catalogueItemType,
+        [AutoData]
+        public static void Validate_DeliveryDateIsValid_IsAdditionalService_DoesNotHaveError(
+            DateTime deliveryDate,
+            CreateOrderItemModel model,
+            CreateOrderItemModelValidator validator)
+        {
+            var recipient = new OrderItemRecipientModel { DeliveryDate = deliveryDate };
+            model.CatalogueItemType = nameof(CatalogueItemType.AdditionalService);
+            model.ServiceRecipients = new List<OrderItemRecipientModel> { recipient };
+
+            var expectedPropertyName =
+                $"{nameof(CreateOrderItemModel.ServiceRecipients)}[0].{nameof(OrderItemRecipientModel.DeliveryDate)}";
+
+            var result = validator.Validate(model);
+            var failure = result.Errors.FirstOrDefault(e => e.PropertyName == expectedPropertyName);
+
+            failure.Should().BeNull();
+        }
+
+        [Test]
+        [AutoData]
+        public static void Validate_DeliveryDateIsNull_IsAdditionalService_HasError(
             CreateOrderItemModel model,
             CreateOrderItemModelValidator validator)
         {
             var recipient = new OrderItemRecipientModel();
-            model.CatalogueItemType = catalogueItemType;
+            model.CatalogueItemType = nameof(CatalogueItemType.AdditionalService);
+            model.ServiceRecipients = new List<OrderItemRecipientModel> { recipient };
+
+            var expectedPropertyName =
+                $"{nameof(CreateOrderItemModel.ServiceRecipients)}[0].{nameof(OrderItemRecipientModel.DeliveryDate)}";
+
+            var result = validator.Validate(model);
+            var failure = result.Errors.FirstOrDefault(e => e.PropertyName == expectedPropertyName);
+
+            Assert.NotNull(failure);
+            failure.ErrorMessage.Should().Be($"{nameof(OrderItemRecipientModel.DeliveryDate)}Required");
+        }
```

Comment:
Can we refactor this test and `Validate_DeliveryDateIsNull_IsSolution_HasError` to a single inline data test based on the catalogue item type?

Reply:
should be able too. will do it now